reward_extraction/behavior_cloning/behavior_cloning.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.functional as F 
+from torch.utils.data import Dataset, DataLoader
+from typing import Tuple
+from robot.data import RoboDemoDset
+from robot.base_robot import XArmTaskSpaceEnv
+# export PYTHONPATH="${PYTHONPATH}:/home/xarm/Documents/JacobAndDavin/test/rewardlearning-robot"
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+class BCNet(nn.Module):
+    def __init__(self) -> None:
+        super(BCNet, self).__init__()
+        self.fc1 = nn.Linear(3, 30)
+        self.fc2 = nn.Linear(30, 300)
+        self.fc4 = nn.Linear(300, 30)
+        self.fc3 = nn.Linear(30, 3)
+        self.relu = nn.LeakyReLU()
+    
+    def forward(self, x):
+        x = self.fc1(x)
+        x = self.relu(x)    
+        x = self.fc2(x)
+        x = self.relu(x)
+        x = self.fc4(x)
+        x = self.relu(x)
+        x = self.fc3(x)
+
+        return x
+    
+
+def data(SCALE) -> Tuple[DataLoader, DataLoader]:
```

Comment:
what's the point of scaling things?

Reply:
we ran into loss = Nan problem (likely) due to gradient explosion, so we were wondering if we scale it by a hundredth when we train and the scale it back when we deploy would resolve the issue. But then we got it to work without scaling , so i think it is unnecessary, will remove after some testing making sure there is no weird bug left unattended. Thanks for pointing it out